### PR TITLE
DE31708 - Update d2l-navigation version

### PR DESCRIPTION
**Pull request by svanherk**
- Includes fixes to the navbar with long course names
- Already included in the cert branch (https://github.com/Brightspace/brightspace-integration/pull/1045), now adding to master

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -212,7 +212,7 @@
     "d2l-more-less": "7fb6369c8792407c6a40aaec8ec47a847f8ef0bb",
     "d2l-my-courses": "63f396bc4f653b76dcc18e7a06552dc2ec4f5189",
     "d2l-my-dashboards": "10b91cbc5118a35675d764208263251573f845e7",
-    "d2l-navigation": "2bdbf66e72c91a1cfe153caa17b55f1f8d077fbe",
+    "d2l-navigation": "c04026b25cea335a0056ac82c408a0562516f0f4",
     "d2l-offscreen": "2c62edef92bb61ede35a93f2ff0660811b666d44",
     "d2l-opt-in-flyout-webcomponent": "827ee03b9a7d3f4a7559a09a077ddf1e774233b5",
     "d2l-organization-hm-behavior": "d1b764b90c349a20aa77c63978f960552882b1a5",
```
Comment (edwardg8) [inline bower.json]: Somehow, I don't see `"lastUpdated"` changed, should be a new timestamp?

Comment (svanherk) [reply bower.json]: I believe Owen removed that to avoid merge conflicts.  Let me see if I can find that comment...

Comment (svanherk) [reply bower.json]: Here: https://github.com/Brightspace/brightspace-integration/pull/1039#issuecomment-429420876

Comment (edwardg8) [reply bower.json]: then the rest looks good to me

**Pull request merged**